PolicyServerElasticsearchContainer/src/main/docker/startup.d/setup-tomcat-ssl-cert.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
This script should already be in the image because it's being copied in from the Tomcat base image.

Reply:
Removed script file